frontend/packages/dev-console/integration-tests/features/helm/install-HelmChart.feature

```diff
@@ -1,84 +1,84 @@
 Feature: Install the Helm Release
     As a user, I want to install the helm release
 
-Background:
-    Given user is at developer perspecitve
-    And user has selected namespace "aut-helm-installation"
+    Background:
+        Given user is at developer perspecitve
```

Comment:
```suggestion
        Given user is at developer perspective
```

Reply:
https://github.com/openshift/console/pull/7656/files address the same

Reply:
Done, https://github.com/openshift/console/pull/7680/commits/9c701865e803bf81296ef5e1eb0f9cf38187dd6d#